OpenQA/Isotovideo/Runner.pm

```diff
@@ -19,6 +23,7 @@ use autotest ();
 use needle ();
 use commands ();
 use distribution ();
+use Mojo::File qw(path);
```

Comment:
Maybe move this up to the other `Mojo::` uses so things are at least a little bit in order/grouped.

Reply:
Done